pkg/storage/jsindexeddb/jsindexeddb.go

```diff
@@ -44,6 +44,16 @@ func (p *Provider) OpenStore(name string) (storage.Store, error) {
 	return &store{name: name, db: db}, nil
 }
 
+// OpenStoreWithDelete open store with Delete() capability
+func (p *Provider) OpenStoreWithDelete(name string) (storage.StoreWithDelete, error) {
```

Comment:
why we need open store with delete

Reply:
I'll remove it..

Reply:
Keys need to be rotated even in the browser. Why would we remove this capability?

Reply:
a store should support Delete() function by default, there's no need to create a `different` store type with Delete() function.

a new patch has been pushed to amend the existing `Store`.

Reply:
Ok - I understand now that the comment was about this particular method